Feature request: Mark tests that need git so they can be disabled on non-git setups

* papis version (`$ papis --version` or commit number): 0.14

# how to reproduce the issue
On a system where `git` isn't installed (for example, a clean chroot on a build system containing the minimal `papis` dependencies), the following tests fail due to its absence
```
FAILED tests/commands/test_git.py::test_git_cli - FileNotFoundError: Command ...
ERROR tests/commands/test_add.py::test_add_run - FileNotFoundError: Command n...
ERROR tests/commands/test_rm.py::test_rm_files_run - FileNotFoundError: Comma...
```

Alternatively, please document that `git` is required to be installed.

--- a/PACKAGING.md
+++ b/PACKAGING.md
@@ -18,7 +18,11 @@ have three numbers, `A.B.C`, where:
 Dependencies
 ------------
 
-See `pyproject.toml` for a complete list of dependencies and minimum versions.
+See `pyproject.toml` for a complete list of Python dependencies and minimum versions.
+We also have some additional optional non-Python dependencies:
+
+* `git`: The Git executable is required for the `papis git` command and other
+  Git integration in various commands.
 
 Wheels
 ------

--- a/tests/commands/test_add.py
+++ b/tests/commands/test_add.py
@@ -1,5 +1,6 @@
 import os
 import pytest
+import shutil
 import sys
 
 import papis.config
@@ -31,6 +32,9 @@ def make_document(name: str, dir: str, nfiles: int = 0) -> papis.document.Docume
     return doc
 
 
+@pytest.mark.skipif(
+    not shutil.which("git"),
+    reason="Test requires 'git' executable to be in the PATH")
 @pytest.mark.library_setup(use_git=True)
 def test_add_run(tmp_library: TemporaryLibrary, nfiles: int = 5) -> None:
     from papis.commands.add import run

--- a/tests/commands/test_git.py
+++ b/tests/commands/test_git.py
@@ -1,10 +1,16 @@
 import os
+import shutil
+
+import pytest
 
 import papis.config
 
 from papis.testing import TemporaryLibrary, PapisRunner
 
 
+@pytest.mark.skipif(
+    not shutil.which("git"),
+    reason="Test requires 'git' executable to be in the PATH")
 def test_git_cli(tmp_library: TemporaryLibrary) -> None:
     from papis.commands.git import cli
     cli_runner = PapisRunner()

--- a/tests/commands/test_rm.py
+++ b/tests/commands/test_rm.py
@@ -1,5 +1,6 @@
 import os
 import pytest
+import shutil
 from _pytest.monkeypatch import MonkeyPatch
 
 import papis.database
@@ -19,6 +20,9 @@ def test_rm_run(tmp_library: TemporaryLibrary) -> None:
     assert not os.path.exists(folder)
 
 
+@pytest.mark.skipif(
+    not shutil.which("git"),
+    reason="Test requires 'git' executable to be in the PATH")
 @pytest.mark.library_setup(use_git=True)
 def test_rm_files_run(tmp_library: TemporaryLibrary) -> None:
     from papis.commands.rm import run

